test/Concurrency/async_task_groups.swift

```diff
@@ -196,7 +196,7 @@ extension Collection {
       var submitted = 0
 
       func submitNext() async throws {
-        await group.add {
+        await group.add { [submitted,i] in
```

Comment:
nice that this is caught now!

Nitpitck:

```suggestion
        await group.add { [submitted, i] in
```

Reply:
Alright, will clean up locally.